### ISSUE TITLE
 making contig depth file...   /home/delaney/software/metaWRAP/bin/metawrap-modules/binning.sh: line 284: jgi_summarize_bam_contig_depths: command not found

### ISSUE THREAD

**Issue by del-oxyribonucleic-acid** [bxlab/metaWRAP#440]
Hello,
I have been trying to run metaWRAP based on work previously done in my lab and I keep getting this error, even when I try to run each binner separately. Any help is much appreciated!

Here is the command and output:
metawrap binning -o ${metawrap_binning_RESULTS} -t 96 -a ${ASSEMBLY} --metabat2 --maxbin2 --concoct ${metawrap_RESULTS}/input_reads/*.fastq

------------------------------------------------------------------------------------------------------------------------
-----                  Aligning 07_metawrap_RESULTS/input_reads/10SPOviralmetaG_clean_1.fastq and                  -----
-----                07_metawrap_RESULTS/input_reads/10SPOviralmetaG_clean_2.fastq back to assembly                -----
------------------------------------------------------------------------------------------------------------------------

[M::mem_pestat] skip orientation FF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (111, 128, 186)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 336)
[M::mem_pestat] mean and std.dev: (136.21, 58.55)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 411)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] skip orientation RR as there are not enough pairs
[main] Version: 0.7.17-r1188
[main] CMD: bwa mem -v 1 -t 96 07_metawrap_RESULTS/binning/work_files/assembly.fa 07_metawrap_RESULTS/input_reads/10SPOviralmetaG_clean_1.fastq 07_metawrap_RESULTS/input_reads/10SPOviralmetaG_clean_2.fastq
[main] Real time: 0.854 sec; CPU: 1.951 sec

------------------------------------------------------------------------------------------------------------------------
-----                               Sorting the 10SPOviralmetaG_clean alignment file                               -----
------------------------------------------------------------------------------------------------------------------------

########################################################################################################################
########################################################################################################################

------------------------------------------------------------------------------------------------------------------------
-----                                          making contig depth file...                                         -----
------------------------------------------------------------------------------------------------------------------------

/home/delaney/software/metaWRAP/bin/metawrap-modules/binning.sh: line 284: jgi_summarize_bam_contig_depths: command not found

************************************************************************************************************************
*****                         Something went wrong with making contig depth file. Exiting.                         *****
************************************************************************************************************************

real    4m58.238s
user    4m57.520s
sys     0m1.546s

**Comment by ursky** [comment]
`jgi_summarize_bam_contig_depths` is a script from metabat2, and it looks like it is not in your PATH. This means either 1) metabat2 is not properly installed or 2) you are not in the right conda environment.

**Comment by wuchch666** [comment]
> So how can I do to run it ? restall metabat2? I have use "binning" successfully in the past. I am convinced that I have use the right conda enviroment .

**Comment by dnolin13** [comment]
yes, I have also successfully binned before using the same script. the jgi script also exists in the conda env so I'm not sure why this error is still happening.

**Comment by OlenaMaiakovska** [comment]
Reinstallation of metabat2 indeed helped.
